services/frontend-service/pkg/handler/environments.go

```diff
@@ -17,28 +17,82 @@ Copyright 2023 freiheit.com*/
 package handler
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
+	"github.com/freiheit-com/kuberpult/pkg/api"
+	"golang.org/x/crypto/openpgp"
+	pgperrors "golang.org/x/crypto/openpgp/errors"
 	"net/http"
+)
 
-	xpath "github.com/freiheit-com/kuberpult/pkg/path"
+const (
+	// This maximum in-memory multipart size.
+	// It was chosen based on the assumption that we have < 10 envs with < 3MB manifests per env.
```

Comment:
I think the comment is misleading, because we only allow creating one environment at a time.